### PR TITLE
License Management (mainly)

### PR DESCRIPTION
**Pull request by robmoffat**

### CODE REVIEW

**Comment by robmoffat** [comment]
Resolving
- #46

**Comment by robmoffat** [comment]
@jstclair2019

**Review by jstclair2019 — APPROVED** [review]
Excellent V1. I had some edits on some of the roles but wanted to get this commit going and we can make adjustments

File: docs/bok/Artifacts/SBOMs.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Software Bill of Materials (SBOM)
+tags: 
+  - OSPO (Role)
+  - Developer (Role)
+---
```
[the remark this answers is not in the document]
Comment (psmulovics) [reply docs/bok/Artifacts/SBOMs.md:6]: Actually that is tomorrow's working group agenda - #95 95

File: docs/osr-resources/oslc-licenses.md [hunk 2]
```diff
@@ -4,7 +4,6 @@ title: OSLC Handbook Licenses
 sidebar_label: Licenses
```
Comment (psmulovics) [inline docs/osr-resources/oslc-licenses.md:4]: We should keep the 'renamed' file instead

File: docs/bok/Activities/License-Management.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+### For Individual Software Projects
+
+Individual software projects often include direct and transitive dependencies on other pieces of open source software.  There are plenty of tools for cataloguing licenses being used:
+
+1. [Scancode](https://github.com/nexB/scancode-toolkit), recommended here: https://softwarerecs.stackexchange.com/questions/40068/tools-to-scan-code-for-open-source-licenses-and-copyrights
+
+2. [License Checker](https://www.npmjs.com/package/license-checker) which is an NPM module you can use in node projects to list the licenses used by the project.
+
+3. [FINOS' Security Scanning](https://github.com/finos/security-scanning#License-reporting-and-scanning) project provides GitHub Actions which will report licenses on Java (Maven), Node and Python projects via the use of [license-maven-plugin,](https://www.mojohaus.org/license-maven-plugin/) [node-license-validator](https://www.npmjs.com/package/node-license-validator) and [pip-licenses](https://pypi.org/project/pip-licenses/) respectively.
```
Comment (psmulovics) [inline docs/bok/Activities/License-Management.md:28]: Move the ',' outside the link

File: docs/bok/Activities/License-Management.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+## License Identification
+
+### For Individual Software Projects
+
+Individual software projects often include direct and transitive dependencies on other pieces of open source software.  There are plenty of tools for cataloguing licenses being used:
+
+1. [Scancode](https://github.com/nexB/scancode-toolkit), recommended here: https://softwarerecs.stackexchange.com/questions/40068/tools-to-scan-code-for-open-source-licenses-and-copyrights
+
+2. [License Checker](https://www.npmjs.com/package/license-checker) which is an NPM module you can use in node projects to list the licenses used by the project.
```
Comment (tsteenbe) [inline docs/bok/Activities/License-Management.md:26]: Can we add [OSS Review Toolkit](https://github.com/oss-review-toolkit/ort) here? This is the tool developed by mostly German automotive OSPOs under the Linux Foundation.

File: docs/bok/Activities/License-Management.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+	
+It is expected that developers would set up these tools to run on the CI/CD pipeline of their projects and generate license reports every time a build is run.
+
+### Across an Estate
+
+Where an organisation is running and managing many projects at once, setting up license scanning on a per-project basis isn't really effective.  Part of the problem is that all the tools listed above have their own output formats.  
+
+Instead, you need tools that will scan open source projects wherever they come from and produce a report in a standard format.  This is called an SBOM.  There are various tools you can use to identify licenses across an estate of projects such as [FOSSA](https://fossa.com), [Mend](https://www.mend.io) and [Snyk](https://snyk.io).
```
Comment (tsteenbe) [inline docs/bok/Activities/License-Management.md:36]: _This is called an SBOM._ - needs better description of SBOM and/or link to SBOM.md.

**Review by tsteenbe — CHANGES_REQUESTED** [review]

File: docs/bok/Activities/License-Management.md (hunk 4 above)
Comment (robmoffat) [reply docs/bok/Activities/License-Management.md:26]: HI @tsteenbe - is the ORT more suitable for the whole-estate section? Seems like this is a great open-source option to add in there.  I might break those out into a list too.

**Comment by robmoffat** [comment]
netlify finally working! https://deploy-preview-89--open-source-readiness.netlify.app/docs/welcome

**Comment by robmoffat** [comment]
@brooklynrob would love your feedback on this

File: docs/bok/Artifacts/SBOMs.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (robmoffat) [reply docs/bok/Artifacts/SBOMs.md:6]: added to our BoK google doc for further discussion

**Review by BrittanyIstenes — APPROVED** [review]
This all looks good!

**Comment by robmoffat** [comment]
Hi,

@psmulovics and I have made a few amendments on https://deploy-preview-89--open-source-readiness.netlify.app/docs/bok/Activities/License-Management 

please review and let us know if we can merge this.  thanks!

**Review by psmulovics — APPROVED** [review]

**Pull request merged**